packages/now-build-utils/test/fixtures/19-yarn-v2/.yarn/releases/yarn-berry.js

```diff
@@ -53,4 +53,4 @@ const r=n(3),A=n(4),i=n(54),o=n(1028),s=process.binding("constants"),a=o(),c="01
  *
  * Copyright © 2015-2018, Jon Schlinkert.
  * Released under the MIT License.
```

Comment:
Can you revert this file? We have lint disabled for the `/test` directory so this shouldn't be modified. Same with `src/main.js` beolow

Reply:
Fixed